docs/6-release-management/6.3.2-helm.md

```diff
@@ -115,6 +113,10 @@ In this exercise we will add a Kubernetes deployment and service to our Helm cha
 
 6. Open the frontend application in a web browser and verify it is communicating with the backend service.
 
+`Challenge:` Can you get the same outcome while ONLY port-forwarding the front-end?
+
+?> Hint: You might have already achieved this depending on how your frontend is communicating with your backend.
```

Comment:
'Hints' like this make me -_-

I would much prefer if we linked service discover docs https://kubernetes.io/docs/concepts/services-networking/service/#discovering-services. I cant recall how we were getting a cluster but I just verified that in a new kind cluster you have CoreDNS